test/base/test_batch.py

```diff
@@ -225,6 +227,21 @@ def test_batch_from_to_numpy_without_copy():
     assert c_mem_addr_new == c_mem_addr_orig
 
 
+def test_batch_copy():
+    batch = Batch(a=[3, 4, 5], b=[4, 5, 6])
```

Comment:
Your test does not make sense. You are dealing with immutable data, so it will be copy when converted in `np.ndarray` by `Batch` constructor in any case. And you should check that `copy=False` is actually not copying the data. To this end, you have to use `batch = Batch(a=np.array([3, 4, 5]), b=np.array([4, 5, 6]))` instead.